.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,73 @@
+name: PerfFlowAspect CI
+
+on: [ push, pull_request ]
+
+jobs:
+
+  build-c:
+    runs-on: ubuntu-18.04
+
+    steps:
+      # Checkout PerfFlowAspect repository under $GITHUB_WORKSPACE
+      - uses: actions/checkout@v2
+
+      - name: Setup Build Env
+        run: |
+          sudo apt-get update
+          sudo apt install clang llvm-dev libjansson-dev libssl-dev bison flex make cmake
+          clang++ --version
+
+      - name: Build Parser
+        run: |
+          cd src/c/parser
+          make VERBOSE=1
+
+      - name: Build Runtime
+        run: |
+          cd src/c/runtime
+          make VERBOSE=1
+
+      - name: Configure Weaver with CMake
+        run: |
+          cd src/c/weaver
+          mkdir build
+          cd build
+          LLVM_DIR=/usr/lib/llvm-6.0/lib/cmake cmake ..
+
+      - name: Build Weaver
+        run: |
+          cd src/c/weaver/build
+          make VERBOSE=1
+
+      - name: Run C Smoke Tests
+        run: |
+          cd src/c/test
+          make
+          sh t0001-cbinding-basic.t
```

Comment:
`t0001-cbinding-basic.t` already include `#! /bin/sh`. Is there a specific reason to include `sh` additionally?

Just so that I understand, if the script execution return non zero, this step will fail in the CI?

Reply:
Good catch, no we do not need to run this script using sh explicitly from the yaml file, we can just launch it with `./` and it behaves just the same. Updated in this PR.